index.js

```diff
@@ -238,6 +253,19 @@ module.exports = function generate(options) {
         });
       });
       metadata.assertionsByType = assertionsByType;
+
+      metadata.assertionsUrl = metadata.collections.assertions.some(
+        page => page.url === '/assertions/'
+      )
+        ? '/assertions/'
+        : assertionsByType[Object.keys(assertionsByType)[0]][0].url;
+
+      metadata.apiPagesUrl = metadata.collections.apiPages.some(
```

Comment:
Assigning `apiPagesUrl` needs to be wrapped in `if (metadata.collections.apiPages.length > 0) { }` otherwise it causes a bad dereference if a plugin has no API documentation (found by linking into unexpected-resemble`).

Reply:
Great thanks for finding that.

Reply:
@alexjeffburke I can't reproduce the problem you are talking about. I'm testing with unexpected-reaction that doesn't have an API section, it even works if I add an `api.md` file. 

I can't build `unexpected-resemble` locally for some reason, so could you try to reproduce the problem with `unexpected-reaction`.

Reply:
Sorry, now I managed to reproduce it. I'll fix it.